magenta/models/nsynth/wavenet/nsynth_generate.py

```diff
@@ -37,6 +38,14 @@
                            "DEBUG, INFO, WARN, ERROR, or FATAL.")
 tf.app.flags.DEFINE_integer("gpu_number", 0,
                             "Number of the gpu to use for multigpu generation.")
+tf.app.flags.DEFINE_integer("num_inter_threads", 0,
+                            "Number of the inter_op_parallelism_threads.")
```

Comment:
I'd be good to mention in the docstring that these only apply to CPU generation.

Reply:
Sure, added the info in the docstring.